### PR TITLE
Added Data Organisation Section to Docs

### PR DESCRIPTION
**Pull request by DrNickClarke**
Docs only

### CODE REVIEW

File: docs/mkdocs/docs/tutorials/data_organisation.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+Often the data is accessed by downstream users and systems much more frequently than it is updated. If that is the case, it is worth organising data to optimise downstream access performance.
+
+### Data Problems: Investigation and Remediation
+
+System problems are often caused by suspected bad data. It is worth thinking about how to investigate and remediate such problems. For example
+
+* Tooling to find and examine suspected bad data
+* Consider how to reconcile suspected bad data vs the original data source
+* Manual overrides to fix bad data to allow the system to run effectively while data is re-sourced
+
```
Comment (athakur91) [inline docs/mkdocs/docs/tutorials/data_organisation.md:81]: is it worth adding a note about how the data versioning makes it easy to restore to the last known good state?

File: docs/mkdocs/docs/tutorials/data_organisation.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+* Some securities may trade less frequently
+
+For many purposes time-aligned data is either necessary or makes the task much simpler.
+
+Resampling is one way to align data. There is a Pandas resample function and a faster function planned in ArcticDB (see [Future Features](#planned-future-features-and-improvements)).
+
+
+## Enterprise Features
+
+Many of the housekeeping and maintenance procedures recommended in this guide are taken care of by processes available in the ArticDB Enterprise package. Please contact us if you would like to explore further.
```
Comment (athakur91) [inline docs/mkdocs/docs/tutorials/data_organisation.md:220]: Please contact us at arcticdb@man.com if you would like to explore further.

File: docs/mkdocs/docs/tutorials/data_organisation.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+
+## Planned Future Features and Improvements
+
+We are always improving and adding new features to ArcticDB. Below are a few relevant new features that are in our plans
+
+* **Multi-Index:** support for more flexible data updates for multi-index symbols.
+* **Resampling:** a planned [QueryBuilder](../api/query_builder) extension that will allow fast on-the-fly resampling of data as part of the read.
+
```
Comment (athakur91) [inline docs/mkdocs/docs/tutorials/data_organisation.md:229]: do we have a process for users to request features? maybe raising an issue / feature request?

**Review by athakur91 — COMMENTED** [review]
Looks good Nick, have added a few comments in line

**Comment by ms041223** [comment]
For the second and third points in the "Data Update Process" section,  its missing capital letters. 
 
- Delivery format (the format that the data source uses) vs storage format of the data. The trade-offs are
- storing data in a format close to delivery format makes it easier to reconcile with the source
- the delivery format is often not the best format for downstream processes
- Frequency of data updates
- Storage growth characteristics over time
- System performance over time

**Comment by ms041223** [comment]
It is really good and will be super useful. Hopefully, this answers many of the questions we have been getting.

File: docs/mkdocs/docs/tutorials/data_organisation.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+* Environment: prod/uat/research
+* Permissions: enviroments that grant/deny read/write permission to different groups of users/systems
+* Accounting: different cloud buckets may be charged to different internal accounts
+* Storage Quotas: different amounts of storage may be allocated for different purposes
+* Storage Performance: faster/slower storage for different applications according to requirements and cost
+
+
+### Library
+
+ArcticDB can handle a large number of libraries per object store - up to 100,000 is perfectly reasonable.
```
Comment (alexowens90) [inline docs/mkdocs/docs/tutorials/data_organisation.md:32]: Have we tested that the performance of this is acceptable for operations like `list_libraries`?

File: docs/mkdocs/docs/tutorials/data_organisation.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+A library groups together a set of related data. Typically its primary purpose is to help catalog the data - well named/documented libraries help people find the data they need.
+
+### Symbol
+
+The symbol is the base unit of storage in ArcticDB. Each symbol is a DataFrame and has optimised indexing on the columns and the index.
```
Comment (alexowens90) [inline docs/mkdocs/docs/tutorials/data_organisation.md:38]: "optimised indexing on the columns and the index" sounds odd

File: docs/mkdocs/docs/tutorials/data_organisation.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+### Initial Data Population
+
+This is usually a one-off process to get a system up and running. Ideally the same scripts can be used for initial data population and regular updates, with different parameters.
+
+### Data Update Process
+
+Regular updates to the data are the norm for most systems. We must consider in our design
+
+* Delivery format (the format that the data source uses) vs storage format of the data. The trade-offs are
+  * storing data in a format close to delivery format makes it easier to reconcile with the source
```
Comment (alexowens90) [inline docs/mkdocs/docs/tutorials/data_organisation.md:64]: Indentation

File: docs/mkdocs/docs/tutorials/data_organisation.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+Please see the [snapshots documentation](snapshots.md) and [snapshots notebook](../notebooks/ArcticDB_demo_snapshots.ipynb) for more details.
+
+
+### Lots of Small Appends or Updates Can Fragment Your Data
+
+Append and Update are efficient because they always add new chunks of data rather than reorganising the existing data (actually update will reorganise but only where necessary which is typically only a small amount).
+
+This means that lots of small appends or updates can result in lots of small data chunks, which makes reads slower.
+
+You can defragment a symbol manually using the [defragment_symbol_data](../api/library.md) library function. Altnernatively the [Enterprise Features](#enterprise-features) offer background processes that will take care of defragmentation for you.
```
Comment (alexowens90) [inline docs/mkdocs/docs/tutorials/data_organisation.md:137]: Altnernatively typo
Also, the other background processes mentioned so far already exist in Man Group, whereas defrag doesn't, and it isn't immediately obvious how to do it safely.

**Review by alexowens90 — CHANGES_REQUESTED** [review]

**Pull request merged**